docs/user-interface/handlers/customize.md

```diff
@@ -61,253 +52,202 @@ public partial class CustomizeEntryPage : ContentPage
-
-To subscribe to, and unsubscribe from, native control events you must register event handlers for the `HandlerChanged` and `HandlerChanging` events on the cross-platform control being customized:
+The handler lifecycle can be used to perform handler customization. For example, to subscribe to, and unsubscribe from, native view events you must register event handlers for the `HandlerChanged` and `HandlerChanging` events on the cross-platform control being customized:
 
 ```xaml
-<Entry HandlerChanged="OnHandlerChanged"
-       HandlerChanging="OnHandlerChanging" />
+<Entry HandlerChanged="OnEntryHandlerChanged"
+       HandlerChanging="OnEntryHandlerChanging" />
 ```
 
-Handlers can be customized per platform by using compiler preprocessor directives, or by using partial classes to organize your code into platform-specific folders and files. Each approach will be discussed in turn, by customizing an `Entry` on Android.
+Handlers can be customized per platform by using conditional compilation, or by using partial classes to organize your code into platform-specific folders and files. Each approach will be discussed in turn, by customizing an `Entry` so that all of its text is selected when it gains focus.
 
-### Using preprocessor directives
+### Conditional compilation
 
-The code-behind file containing the event handlers for the `HandlerChanged` and `HandlerChanging` events is shown in the following example, which uses preprocessor directives:
+The code-behind file containing the event handlers for the `HandlerChanged` and `HandlerChanging` events is shown in the following example, which uses conditional compilation:
 
 ```csharp
-using Microsoft.Maui.Platform;
+#if ANDROID
+using AndroidX.AppCompat.Widget;
+#elif IOS || MACCATALYST
+using UIKit;
+#elif WINDOWS
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+#endif
 
-namespace CustomizeHandlersDemo;
+namespace CustomizeHandlersDemo.Views;
 
-public partial class CustomizeEntryPage : ContentPage
+public partial class CustomizeEntryHandlerLifecyclePage : ContentPage
 {
-    public CustomizeEntryPage()
+    public CustomizeEntryHandlerLifecyclePage()
     {
         InitializeComponent();
     }
 
-    void OnHandlerChanged(object sender, EventArgs e)
+    void OnEntryHandlerChanged(object sender, EventArgs e)
     {
 #if ANDROID
-        ((sender as Entry).Handler.PlatformView as Android.Views.View).FocusChange += OnFocusChange;
+        ((sender as Entry).Handler.PlatformView as AppCompatEditText).SetSelectAllOnFocus(true);
```

Comment:
This code causes the code box on the published page to scroll right. You could avoid that by casting at the start of the method
```csharp
Entry entry = sender as Entry;
```